R/exclusions.R

```diff
@@ -38,7 +38,7 @@ exclude_from_title <- function(data) {
     # Items related to dissemination
     exclude(title, "open( |-|, )(access|review(er)?)|publishing|communication") |>
     # Items related to education
-    exclude(title, "open( |-|, )(textbooks?|doors?|education(al)?|seminar|learning|schools|course|book|pedagogy)") |>
+    exclude(title, "open( |-|, )(textbooks?|doors?|education(al)?|seminar|learning|schools|course|book|pedagogy|teaching|learning|online?)") |>
```

Comment:
```suggestion
    exclude(title, "open( |-|, )(textbooks?|doors?|education(al)?|seminar|learning|schools|course|book|pedagogy|teaching|learning|online)") |>
```

Reply:
I think those are great!! I just asked chatGPT to get me from a sample of 200 titles, the 5 main topics across the titles and, from those, the top 5 key words. A whole section was about education and all the changes here were included. Let's do this!!